sysutils/tmate/Portfile

```diff
@@ -22,7 +22,7 @@ depends_build       port:pkgconfig
 
 depends_lib         port:libevent \
                     port:ncurses \
-                    port:msgpack1 \
+                    port:msgpack \
```

Comment:
changing a dependency requires to increase the "revision" so that the package will be rebuild for everyone with this new dependency